docs/valuesYaml-valuesFiles.md

```diff
@@ -0,0 +1,64 @@
+# Understanding Helm values.yaml vs. Fleet valuesFiles
+
+When using Fleet with Helm, you often confuse the chart’s built-in `values.yaml` with additional values files referenced in `fleet.yaml`. These files serve different purposes, and it’s important to understand how they interact.
```

Comment:
This sounds a bit too direct to me, as if we were blaming users for something. How about something more positive, such as:
```suggestion
Installing Helm charts with Fleet offers multiple ways of configuring and referencing values, using the chart’s built-in `values.yaml` and additional values files referenced in `fleet.yaml`. These files serve different purposes, and it’s important to understand how they interact.
```